### PR TITLE
Add v2 query range metrics API

### PR DESCRIPTION
**Pull request by srikanthccv**

### CODE REVIEW

File: frontend/src/constants/env.ts [hunk 1]
```diff
@@ -1,6 +1,3 @@
 export const ENVIRONMENT = {
-	baseURL:
```
Comment (palash-signoz) [inline frontend/src/constants/env.ts:2]: @srikanthccv please remove this change

**Review by palash-signoz — CHANGES_REQUESTED** [review]

File: frontend/src/constants/env.ts (hunk 1 above)
Comment (srikanthccv) [reply frontend/src/constants/env.ts:2]: Yes, I was going to undo this. Just wanted to let others review the work so far.

File: pkg/query-service/app/http_handler.go [hunk 2]
```diff
@@ -302,18 +303,84 @@ func (aH *APIHandler) metricAutocompleteTagValue(w http.ResponseWriter, r *http.
...
+	// build queries
+	queries, err := metricsQueryRangeParams.BuildQuery("time_series")
+	if err != nil {
+		aH.respondError(w, &model.ApiError{Typ: model.ErrorBadData, Err: err}, nil)
 	}
-	aH.respond(w, response_data)
+
+	var formulaSubQuery string
+	var results []*[]model.MetricResult
+	// read the result for individual metric queries and build query for formula
```
Comment (mindhash) [inline pkg/query-service/app/http_handler.go:320]: @srikanthccv can we build the entire result in one function, so I can re-use it in alerts. http handler methods will not be callable from alerting module. 

Ideally (reader *clickhouseReader) BuildMetricQueryResult() (result interface{}, err error) or something that I could call from alerts will be really useful

File: frontend/src/constants/env.ts (hunk 1 above)
Comment (palash-signoz) [reply frontend/src/constants/env.ts:2]: cool ..

File: pkg/query-service/app/http_handler.go (hunk 2 above)
Comment (srikanthccv) [reply pkg/query-service/app/http_handler.go:320]: Makes sense.

**Review by ankitnayan — APPROVED** [review]

**Pull request merged**